src/linalg.jl

```diff
@@ -56,7 +56,11 @@ end
 # Transpose, conjugate, etc
 @inline conj(a::StaticArray) = map(conj, a)
 @inline transpose(m::StaticMatrix) = _transpose(Size(m), m)
-# note: transpose of StaticVector is a Adjoint, handled by Base
+# note: transpose of StaticVector is a Transpose, handled by Base
+if VERSION >= v"0.7-"
+    @inline transpose(a::Adjoint{<:Any,<:Union{StaticVector,StaticMatrix}}) = conj(a.parent)
```

Comment:
Slightly surprised that this isn't done in `Base`. @mbauman?

Reply:
`conj` is eager, while `transpose` and `adjoint` are generally lazy. (Yes, StaticArrays should probably also switch to supporting lazy `transpose`/`adjoint`, but let's take one step at the time).

Reply:
That’s right, I remember now. I did try to sneak in lazy `conj` but it never happened.